src/apps/sommelier/positions.ts

```diff
@@ -0,0 +1,101 @@
+import { logger } from '../../log'
+import { getClient } from '../../runtime/client'
+import { getTokenId } from '../../runtime/getTokenId'
+import { toDecimalNumber } from '../../types/numbers'
+import {
+  PositionDefinition,
+  PositionsHook,
+  PricePerShareContext,
+  UnknownAppTokenError,
+} from '../../types/positions'
+import { cellarV0821Abi } from './abis/cellar'
+import { getSommStrategiesData } from './sommelierApi'
+
+const hook: PositionsHook = {
+  getInfo() {
+    return {
+      id: 'sommelier',
+      name: 'Sommelier',
+      description: 'Sommelier vaults',
+    }
+  },
+  async getPositionDefinitions({ networkId }) {
```

Comment:
ah, thanks for explaining! i was confused about how to use `address` before, since the actual balance was being added in the runtime i didn't think further on the topic. i added a check for the balance in the hook itself, but i agree at some point it might be nice for it to be in the runtime